src/vm-repair/azext_vm_repair/_validators.py

```diff
@@ -0,0 +1,199 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from datetime import datetime
+from json import loads
+from re import match, search
+from knack.log import get_logger
+from knack.util import CLIError
+
+from azure.cli.command_modules.vm.custom import get_vm, _is_linux_os
+from azure.cli.command_modules.resource._client_factory import _resource_client_factory
+from msrestazure.azure_exceptions import CloudError
+from msrestazure.tools import parse_resource_id, is_valid_resource_id
+
+from .exceptions import AzCommandError
+from .repair_utils import _call_az_command, _get_repair_resource_tag, _uses_encrypted_disk
+
+# pylint: disable=line-too-long
+
+logger = get_logger(__name__)
+
+def validate_create(cmd, namespace):
+
+    # Check if VM exists and is not classic VM
+    source_vm = _validate_and_get_vm(cmd, namespace.resource_group_name, namespace.vm_name)
+    is_linux = _is_linux_os(source_vm)
+
+    # Check repair vm name
+    if namespace.repair_vm_name:
+        _validate_vm_name(namespace.repair_vm_name, is_linux)
+    else:
+        namespace.repair_vm_name = ('repair-' + namespace.vm_name)[:15]
+
+    # Check copy disk name
+    timestamp = datetime.now().strftime('%Y%m%d%H%M%S')
```

Comment:
expected local datetime of server or can this be UTC datetime?

Reply:
Fixed to datetime.utcnow().strftime('%Y%m%d%H%M%S'). Good catch.